src/wagtail_personalisation/admin.py

```diff
@@ -2,7 +2,7 @@
 
 from django.contrib import admin
 
-from wagtail_personalisation import models, rules
+from . import models, rules
```

Comment:
I'm not really a fan of relative imports, it always requires me to visualise the directory tree to see where files are imported from. Can we keep them absolute?